JustSaying/JustSayingBus.cs

```diff
@@ -158,33 +161,42 @@ private IMessagePublisher GetActivePublisherForMessage(Message message)
                 throw new InvalidOperationException(errorMessage);
             }
 
-            string activeRegion;
-            if (Config.GetActiveRegion == null)
+            var activeRegion = GetActiveRegion();
+            if (!string.Equals(_previousRegion, activeRegion, StringComparison.Ordinal))
             {
-                activeRegion = Config.Regions.First();
+                _log.LogInformation("Active region for publishing has been set to '{Region}', was '{PreviousRegion}'.",
```

Comment:
I think `{ActiveRegion}` would be a better name here to contrast with `{PreviousRegion}`.

Reply:
We use just `{Region}` in most places.